integration/basic_test.go

```diff
@@ -472,6 +509,25 @@ func TestDatabaseName(t *testing.T) {
 				},
 				altMessage: fmt.Sprintf("Invalid namespace: %s.%s", dbName64, "TestDatabaseName-Err"),
 			},
+			"WithASlash": {
+				db: "/",
+				err: &mongo.CommandError{
+					Name:    "InvalidNamespace",
+					Code:    73,
+					Message: `Invalid namespace specified '/.TestDatabaseName-Err'`,
+				},
+				altMessage: `Invalid namespace: /.TestDatabaseName-Err`,
+			},
+
+			"WithABackslash": {
+				db: "\\",
+				err: &mongo.CommandError{
+					Name:    "InvalidNamespace",
+					Code:    73,
+					Message: `Invalid namespace specified '\.TestDatabaseName-Err'`,
+				},
+				altMessage: `Invalid namespace: \.TestDatabaseName-Err`,
+			},
```

Comment:
Could you test database name with number prefix please. Something like `0` is great :hugs:

Reply:
As we don't support number prefixes for postgres, the test will fail for pg.